fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/CredentialsService.java

```diff
@@ -25,11 +24,11 @@
 public interface CredentialsService {
 
     /**
-     * Get the credentials for the given servlet request
+     * Get the credentials for the given request
      *
-     * @param request the servlet request
+     * @param request the request
      * @return the JCR credentials for the given request
      */
-    Credentials getCredentials(final HttpServletRequest request);
+    Credentials getCredentials(final Object request);
```

Comment:
I have no objection-- just wanted to make sure I understood the move correctly. This is better than what we have and blocks no future moves to get even better, so 👍 .